src/GovUk.Education.ExploreEducationStatistics.Notifier.Model/PublicationNotificationMessage.cs

```diff
@@ -1,16 +1,27 @@
+#nullable enable
 using System;
 
 namespace GovUk.Education.ExploreEducationStatistics.Notifier.Model
 {
     public class PublicationNotificationMessage
     {
         public Guid PublicationId { get; set; }
-        public string Name { get; set; }
-        public string Slug { get; set; }
+        public string PublicationName { get; set; } = "";
+        public string PublicationSlug { get; set; } = "";
+
+        public string ReleaseName { get; set; } = "";
+        public string ReleaseSlug { get; set; } = "";
+
+        public bool Amendment { get; set; }
+        public string UpdateNote { get; set; } = "";
 
         protected bool Equals(PublicationNotificationMessage other)
         {
-            return Name == other.Name && PublicationId.Equals(other.PublicationId) && Slug == other.Slug;
+            return PublicationId.Equals(other.PublicationId)
```

Comment:
I'm assuming we are overriding Equals/Hashcode to facilitate easier comparison during unit testing?

I think you could make this a `record` type in which case you won't have to do this as two records objects are the same if they store the same values. Then you wouldn't need to worry about which fields are included. See other comments about `UpdateNote` being included here which means it's not being unit tested correctly in the current code.

The setters can also be made `init` as they should never be updated.

Reply:
Switched PublicationNotificationMessage from a class to a record